src/Symfony/Component/Validator/Constraints/ExpressionValidator.php

```diff
@@ -65,13 +68,87 @@ public function validate($value, Constraint $constraint)
 
     private function getExpressionLanguage()
     {
-        if (null === $this->expressionLanguage) {
+        if (!$this->expressionLanguage instanceof ExpressionLanguage) {
```

Comment:
This is a bug fix on 4.4, `$this->expressionLanguage` can be anything. I will extract it in a separate PR if this one doesn't end up being merged.

Reply:
Fixed in https://github.com/symfony/symfony/pull/33831

Reply:
Thanks, reverting here then.